linux-pull-run.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+while getopts 'o' OPTION; do
+    case "$OPTION" in
+        o)
+            no_open=true
+            ;;
+        ?)
+            echo "script usage: ./linux-pull-run.sh [-o]" >&2
+            echo $'\t-o: disables automatic opening of the browser'
+            exit 1
+            ;;
+    esac
+done
+docker pull ghcr.io/badgerloop-software/chase-car-dashboard-image:latest
+if [[ ! $no_open ]]; then
+    [[ $OSTYPE != 'darwin'* ]] && cmd=xdg-open
+    [[ $OSTYPE == 'darwin'* ]] && cmd=open
+    if  [[-n $($cmd 2>/dev/null) ]]; then
```

Comment:
```suggestion
    if [[ -n $($cmd 2>/dev/null) ]]; then
```
Bash can be picky about whitespace

Reply:
fixed!